### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by Stephanie-Delicia**
Please review the login screen and let me know if anything is needed. The file to review is LoginScreen.kt which contains the composable.

### CODE REVIEW

File: app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                    .height(50.dp)
+                    .width(250.dp)
+                    .constrainAs(password) {
+                        top.linkTo(username.bottom, margin = (maxHeight/20).dp)
+                        absoluteLeft.linkTo(
+                            parent.absoluteLeft,
+                            margin = (maxWidth/2 - 250/2).dp)
+                    })
+
+            // Button for viewing my followers
```
Comment (tomgrbz) [inline app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt]: typo in comment from maybe a copy/paste?

Comment (Stephanie-Delicia) [reply app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt]: Yes! I can remove that.

File: app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+
+private val darkerBlue :  androidx.compose.ui.graphics.Color = Color(0xFF6880FF) // 0xFF9BAAF8
```
Comment (tomgrbz) [inline app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt]: Thinking just from how they've graded things in the past, but might want to put all of these colors in the colors.xml or a theme maybe, but im not too sure on how much that requires

Comment (tomgrbz) [reply app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt]: Other than that, it looks amazing Stephanie!!

**Review by tomgrbz — APPROVED** [review]

**Comment by tomgrbz** [comment]
Actually one thing is when branching off of this, I had to change an import and add a dependency for the `Constraint Layout` composable:


import androidx.constraintlayout.compose.ConstraintLayout
 implementation("androidx.constraintlayout:constraintlayout:2.1.4")
 
 Unless im missing something, but this is after pulling most recent main

**Comment by Stephanie-Delicia** [comment]
> Actually one thing is when branching off of this, I had to change an import and add a dependency for the `Constraint Layout` composable:
> 
> import androidx.constraintlayout.compose.ConstraintLayout implementation("androidx.constraintlayout:constraintlayout:2.1.4")
> 
> Unless im missing something, but this is after pulling most recent main

Before merging, I pulled on this branch, then switched to main and pulled again (so I _think_ I was up to date). I only had an error asking for an import for navigation-compose and was able to run the app after. I am sorry, I saw this comment after pushing, should I change the import on main?

**Comment by tomgrbz** [comment]
> > Actually one thing is when branching off of this, I had to change an import and add a dependency for the `Constraint Layout` composable:
> > import androidx.constraintlayout.compose.ConstraintLayout implementation("androidx.constraintlayout:constraintlayout:2.1.4")
> > Unless im missing something, but this is after pulling most recent main
> 
> Before merging, I pulled on this branch, then switched to main and pulled again (so I _think_ I was up to date). I only had an error asking for an import for navigation-compose and was able to run the app after. I am sorry, I saw this comment after pushing, should I change the import on main?

Not a major issue ;) so we should be fine.

**Pull request merged**